ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Node/Praos.hs

```diff
@@ -114,14 +118,12 @@ praosSharedBlockForging
   ProtocolInfo
 -------------------------------------------------------------------------------}
 
--- | Parameters needed to run Babbage
-data ProtocolParamsBabbage c = ProtocolParamsBabbage {
+data instance ProtocolParams (ShelleyBlock (Praos c) (BabbageEra c)) = ProtocolParamsBabbage {
```

Comment:
I should probably put the constructor on the next line. Idem for all the other `ProtocolParams` instances.